locale/en/corteza-webapp-compose/module.yaml

```diff
@@ -9,6 +9,8 @@ allRecords:
   columns:
     title: Configure columns
     description: The columns picked won't be saved after you refresh the page
+  tooltip:
+    configureColumns: Selecting more than 10 columns and/or fields of type user or record will impact performance
```

Comment:
This has to be reworded since it conveys the wrong message.
`Selecting a large number of fields or displaying record/user fields will impact performance` for example

Same in the upper one, tho Im pretty sure you just need one translation